core/src/main/java/tech/pegasys/web3signer/core/multikey/DefaultArtifactSignerProvider.java

```diff
@@ -45,7 +44,7 @@ public DefaultArtifactSignerProvider(
   public void reload() {
     LOG.trace("Reloading Artifact Signers");
 
-    final Map<String, ArtifactSigner> signerMap =
+    signers =
```

Comment:
Couldn't this result in some keys being removed?

Reply:
nice catch ... fixed.